docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,92 @@
+{{% /alert %}}
+
+
+If you created a cluster without the package controller or if the package controller was not properly configured, you may need to do some things to enable it.
+
+Make sure you are authenticated with the AWS CLI. Use the credentials you set up for packages. These credentials should have [limited capabilities]({{< relref "./prereq#setup-authentication-to-use-curated-packages" >}}):
+
+```bash
+export AWS_ACCESS_KEY_ID="your*access*id"
+export AWS_SECRET_ACCESS_KEY="your*secret*key"
+export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
+export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"
+```
+
+Verify your credentials are working:
+```shell
+aws sts get-caller-identity
+```
+Login to docker
+```bash
+aws ecr get-login-password |docker login --username AWS --password-stdin 783794618700.dkr.ecr.us-west-2.amazonaws.com
+```
+
+Verify you can pull an image
+```bash
+docker pull 783794618700.dkr.ecr.us-west-2.amazonaws.com/emissary-ingress/emissary:v3.5.1-bf70150bcdfe3a5383ec8ad9cd7eea801a0cb074
+```
+If the image downloads successfully, it worked!
+
+If you do not have the package controller installed (it is installed by default), install it now:
+```shell
+eksctl anywhere install packagecontroller -f cluster.yaml
+```
+If you had the package controller disabled, you may need to modify your `cluster.yaml` to enable it.
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: billy
+spec:
+  packages:
+    disable: false
+```
+
+You may need to create or update your credentials which you can do with a command like this. Set the environment variables to the proper values before running the command.
+```bash
+kubectl delete secret -n eksa-packages aws-secret
+kubectl create secret -n eksa-packages generic aws-secret \
+   --from-literal=AWS_ACCESS_KEY_ID=${EKSA_AWS_ACCESS_KEY_ID} \
+   --from-literal=AWS_SECRET_ACCESS_KEY=${EKSA_AWS_SECRET_ACCESS_KEY}  \
+   --from-literal=REGION=${EKSA_AWS_REGION}
+```
+
+### Upgrade the packages controller
+
+{{% alert title="Important" color="warning" %}}
+EKS Anywhere v0.15.0 (packages controller v0.3.9+) and onwards includes support for eks-anywhere-packages controller as a self-managed package feature. The package controller now  upgrades automatically according to the version within the selected package bundle. For any version prior to v0.3.X, manual steps must be executed to upgrade.
+{{% /alert %}}
+
+To manually upgrade the package controller,
+
+1. Ensure the namespace will be kept
+```
+kubectl annotate namespaces eksa-packages helm.sh/resource-policy=keep
+```
+
+2. Uninstall the eks-anywhere-package helm release
+```
+helm uninstall eks-anywhere-packages
+```
+
+3. Remove the secret called aws-secret (we will need credentials when installing the new version)
+```
+kubectl delete secret -n eksa-package aws-secret
+```
+
+4. Install the new version using the latest eksctl-anywhere binary
+```
+eksctl anywhere install packagecontroller -f ${CLUSTER_NAME}.yaml
```

Comment:
Is this the management or workload CLUSTER_NAME?

Reply:
The management cluster. I will clarify that